<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QLineEdit" name="line_source">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Source</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_destination">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_resolution">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>32</number>
    </property>
    <property name="maximum">
     <number>1024</number>
    </property>
    <property name="value">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_azimuth">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>45</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_zenith">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="value">
     <number>45</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_res">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_azimuth">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Azimuth</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zenith">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zenith</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_position">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_overscan">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Overscan</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_sample">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>36</number>
    </property>
    <property name="value">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_sample">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Samples</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton_render">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RENDER</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolbutton_source">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>24</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolbutton_destination">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>24</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_useDest">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>79</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Use</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_fov">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_fov">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FOV</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolbutton_lights">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>24</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_useLights">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>79</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Use</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_lights">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>.lights.ass</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="box_position">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.250000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="box_overscan">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.250000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_imagebox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuassThumb">
    <property name="title">
     <string>assThumb</string>
    </property>
    <widget class="QMenu" name="menuDefaults">
     <property name="title">
      <string>Defaults</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionSave"/>
     <addaction name="actionLoad"/>
    </widget>
    <addaction name="menuDefaults"/>
    <addaction name="actionExport_Expanded"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuassThumb"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport_Expanded">
   <property name="text">
    <string>Export Expanded</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
